src/parse.rs

```diff
@@ -204,7 +234,7 @@ enum ItemBody {
 
     Rule,
     Heading(u32), // heading level
-    FencedCodeBlock(CowIndex),
+    FencedCodeBlock(CowIndex, CowIndex),
```

Comment:
The fence is still stored here, which is unnecessary and increases the size of a `Node<Item>` by a word.